README.md

```diff
@@ -1 +1,230 @@
-# ballerina-aks-deployment
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/ballerina-aks-deployment.svg?branch=master)](https://travis-ci.org/ballerina-guides/ballerina-aks-deployment)
+
+# Deployment Ballerina with Azure Kubernetes Service
+
+[AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/) provides an easy to use hosted Kubernetes cluster service, which drastically reduces your time to setup your own infrastructure.
```

Comment:
Let's introduce the AKS abbreviation properly here and say " Azure Kubernetes Service (AKS) provides..."

Reply:
Sure, thanks.